source/jormungandr/jormungandr/interfaces/v1/ResourceUri.py

```diff
@@ -96,6 +96,14 @@ def get_filter(self, items, args):
                 else:
                     filter_list.append(type_ + ".uri=" + protect(item))
                 type_ = None
+        # handle tags
+        tags = args.get("tags[]", [])
+        if tags:
+            tag_list = []
+            for tag in tags:
+                tag_list.append("disruption.tag({})".format(protect(tag)))
+            tag_filter = '({})'.format(" or ".join(tag_list))
+            filter_list.append(tag_filter)
         return " and ".join(filter_list)
```

Comment:
parenthesis everywhere would be safer in my opinion:
```python
return '({})'.format(') and ('.join(filter_list))
```

This way you protect anything and don't need to change l.69